app/actions/playlists.js

```diff
@@ -29,20 +30,38 @@ export function addPlaylist(tracks, name) {
   };
 }
 
+export function deletePlaylist(name) {
+  return dispatch => {
+    let playlists = store.get('playlists') || {};
+    if (playlists) {
```

Comment:
Instead of manually filtering `playlists` like this, you can do:
```javascript
playlists = _.filter(playlists, item => item.name !== name);
```
This will filter out the entry with the name from the argument from the playlist.

Reply:
It will return a json object which matches. To delete the given playlist from the array I need to get the index of the playlist.

Reply:
done